### ISSUE TITLE
How to ctreat your own loss fuction in which the weights and output of hiden layer has been used? 

### ISSUE THREAD

**Issue by huzhengwei1993** [keras-team/keras#11135]
for example:

def my_loss(d,a1,a2,l,alpha):
    return linalg.norm(a2-d)+linalg.norm(a1.T@l@a1)

model=Sequential()
model.add(Dense(100, activation='relu',name='layer1'))
model.add(Dense(1000 activation=‘relu’,use_bias=False,name='layer2'))
model.compile(optimizer='rmsprop',loss=my_loss,metrics=['accuracy'])

### alpha is a constant

How do I achieve the above functions?
（Note: The code is just for describing problems only. It can not be run!）

**Comment by harshini-gadige** [comment]
Hi

This issue isn't related to a bug/enhancement/feature-request or other accepted types of issue.

To ask questions, please see the following resources :

https://keras.io/
https://keras.io/#support
https://gitter.im/Keras-io/Lobby

Thanks!
